content/intro-to-storybook/ember/en/screen.md

```diff
@@ -144,40 +185,51 @@ We also change the `application` template to render the `InboxScreen` (eventuall
 
 However, where things get interesting is in rendering the story in Storybook.
 
-As we saw previously, the `TaskList` component is a **container** that renders the `PureTaskList` presentational component. By definition with other frameworks, container components cannot be simply rendered in isolation; they expect to be passed some context or to connect to a service.
-
-When placing the `TaskList` into Storybook, we were able to ilustrate this issue by simply rendering the `PureTaskList` and avoiding the container. We'll do something similar and render the `PureInboxScreen` in Storybook also.
+As `loading` and `error` are states internal to the `InboxScreen` component,
+they usually aren't controlled from the outside, so we allow those to be passed
+in as arguments. That will enable us to showcase these variations in Storybook
+and enable design review during onto them.
 
-So when we setup our stories in `pure-inbox-screen.stories.js`:
+So when we setup our stories in `inbox-screen.stories.js`:
 
 ```javascript
-// app/components/pure-inbox-screen.stories.js
-import { hbs } from 'ember-cli-htmlbars';
+// app/components/inbox-screen.stories.js
+import { hbs } from "ember-cli-htmlbars";
 
 export default {
-  title: 'PureInboxScreen',
-  component: 'pure-inbox-screen',
+  title: "InboxScreen",
+  component: "InboxScreen",
 };
 
 export const Default = () => ({
-  template: hbs`<PureInboxScreen/>`,
+  template: hbs`<InboxScreen/>`,
 });
 
-export const error = () => ({
-  template: hbs`<PureInboxScreen @error={{true}}/>`,
-});
+export const Error = () => {
+  return {
+    template: hbs`<InboxScreen @error={{true}}/>`
+  }
+};
+
+export const Loading = () => {
+  return {
+    template: hbs`<InboxScreen @loading={{true}}/>`
+  }
+}
 ```
 
-We see that both the `error` and `standard` stories work just fine. (But you will encounter some problems when trying to test the `PureInboxScreen` with a unit test if no data is supplied like we did with `TaskList`).
+We see that both the `Error`, `Loading` and `Default` stories work just fine.
 
 <div class="aside">
-As an aside, passing data down the hierarchy is a legitimate approach, especially when using <a href="http://graphql.org/">GraphQL</a>. It’s how we have built <a href="https://www.chromatic.com">Chromatic</a> alongside 800+ stories.
+Traditionally in ember, routes do support `loading` and `error` states. Though the motivation
```

Comment:
@gossi if you don't mind capitalize the framework's name if you don't mind. It should be Ember